benchmark_runner/common/oc/oc.py

```diff
@@ -411,11 +411,28 @@ def verify_odf_installation(self, namespace: str = 'openshift-storage'):
 
     def get_odf_disk_count(self):
         """
-        This method returns odf disk count
-        :return:
+        This method returns the ODF disk count.
+        :return: ODF disk count or -1 if the count cannot be retrieved
         """
         if self.is_odf_installed():
-            return int(self.run(f"{self.__cli} get --no-headers pod -n openshift-storage | grep osd | grep -cv prepare"))
+            try:
+                # Run the command to get ODF disk count
+                disk_count_str = self.run(
+                    f"{self.__cli} get --no-headers pod -n openshift-storage | grep osd | grep -cv prepare")
+                disk_count = int(disk_count_str)
+                return disk_count
+            except ValueError as e:
+                # Log the error and return -1 as a fallback
+                logger.error(f"Error converting ODF disk count to integer: {e}")
+                return -1
+            except Exception as e:
```

Comment:
I know this usually won't be run interactively, but it isn't a bad idea to catch KeyboardInterrupt and BrokenPipe and exit.

Reply:
'-1' presents the number of odf disks